locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
+	"View": "Wiedergabe",
+	"Reload": "Neu laden",
+	"Force Reload": "Neu laden // erzwingen?",
+	"Toggle Developer Tools": "Entwicklerwerkzeuge umschalten",
+	"Actual size": "Wahre Größe",
+	"Zoom in": "Hineinzoomen",
+	"Zoom out": "Rauszoomen",
+	"Toggle fullscreen": "Vollbild umschalten",
+	"Window": "Fenster",
+	"Minimize": "Minimieren",
+	"Close": "Schliessen",
+	"FreeTube": "FreeTube",
+	"Subscriptions": "Abonnements",
+	"Featured": "Vorgestellt",
+	"Most Popular": "Beliebteste ?",
+	"Saved": "Gespeichert",
+	"Playlists": "Playlists",
+	"History": "Historie",
+	"Settings": "Einstellungen",
+	"About": "Über",
+	"Search / Go to URL": "Suche / URL besuchen",
+	"Search Results": "Suchergebnisse",
+	"Subscriber": "Subscriber",
+	"Subscriber": "Subscribers",
+	"Video": "Video",
+	"Videos": "Videos",
+	"View Full Playlist": "Vollständige Playlist anzeigen",
+	"Live Now": "Jetzt Live",
+	"Fetch more results": "Mehr Ergebnisse anzeigen",
+	"Fetching results. Please wait": "Ergebnisse laden. Bitte warten",
+	"Latest Subscriptions": "Neueste Abonnements",
+	"Save Video": "Video Speichern",
+	"Remove Saved Video": "Gescpeichertes Video entfernen // ?",
+	"Open in YouTube": "Öffnen in YouTube",
+	"Copy YouTube Link": "YouTube Link kopieren",
+	"Open in HookTube": "Öffnen in HookTube",
+	"Copy HookTube Link": "HookTube Link kopieren",
+	"URL has been copied to the clipboard": "Die URL wurde kopiert // clipboard not known",
```

Comment:
`Die URL wurde in die Zwischenablage kopiert`

Reply:
Thank you.